ios/RNDateTimePickerManager.m

```diff
@@ -40,4 +40,10 @@ - (UIView *)view
 RCT_REMAP_VIEW_PROPERTY(mode, datePickerMode, UIDatePickerMode)
 RCT_REMAP_VIEW_PROPERTY(timeZoneOffsetInMinutes, timeZone, NSTimeZone)
 
+RCT_CUSTOM_VIEW_PROPERTY(textColor, UIColor, RNDateTimePicker)
+{
+  [view setValue:[RCTConvert UIColor:json] forKey:@"textColor"];
```

Comment:
this function needs to be ready to be able to cope with `json` representing both the case when textColor is `ColorValue` and when it's undefined / null in which case it should use default color.